docs/getting_started.md

```diff
@@ -1,8 +1,11 @@
-This guide will help you get started with the Mindee Ruby  OCR SDK to easily extract data from your documents.
+---
+title: Ruby Getting Started
+---
+This guide will help you get the most out of the Mindee Ruby client library to easily extract data from your documents.
 
-The Ruby client supports [Invoice](https://developers.mindee.com/docs/ruby-invoice-ocr), [receipt](https://developers.mindee.com/docs/ruby-receipt-ocr),  [passport](https://developers.mindee.com/docs/ruby-passport-ocr),  OCR APIs and [custom-built API](https://developers.mindee.com/docs/ruby-api-builder) from the API Builder.
-
-You can view the source code on [GitHub](https://github.com/mindee/mindee-api-ruby).
+> 📘 **Info**
+> 
+> The library is typed-hinted using sorbet.
```

Comment:
I'm not seeing the sorbet dependencies in the gemfile, and don't see any signature specs in ruby files.
So doesn't look like this is actually true...

Reply:
Yeah I think I hallucinated that one hard :laughing: